### ISSUE TITLE
new release

### ISSUE THREAD

**Issue by DanielRuf** [jamestalmage/cli-table2#43]
Please make a new release and push to npmjs.com.

**Comment by omarestrella** [comment]
Any plans or updates concerning a release? The library hasnt been published in 2 years and there have been significant changes since then (specifically the removal of the lodash dependency).

**Comment by DanielRuf** [comment]
https://github.com/jamestalmage/cli-table2/releases
https://www.npmjs.com/package/cli-table2

**Comment by DanielRuf** [comment]
https://github.com/jamestalmage/cli-table2/compare/v0.2.0...master

**Comment by ronkorving** [comment]
@jamestalmage Please? People are waiting for this for a while now :'(

**Comment by taichi** [comment]
i made handy release for this issue. https://www.npmjs.com/package/@taichi/cli-table2
but i want to use official release.

**Comment by DanielRuf** [comment]
@jamestalmage let us know if and we can help you here or if you need some more maintainers =)

**Comment by schalkneethling** [comment]
Agreed, we are also running into this on a few repos, for example:
https://github.com/mozilla/kuma/pull/4768

More than happy to help out with getting a release out there. Perhaps we can integrate semantic-release[https://github.com/semantic-release/semantic-release] to aid here? Happy to open a PR to add that. Let me know @jamestalmage

**Comment by Turbo87** [comment]
maybe we should fork this into `cli-table3`... 🙈

**Comment by schalkneethling** [comment]
@Turbo87 that is exactly what I was thinking. I was thinking of doing it myself but, there are unanswered questions that needs answering.

How much maintenance will be required for whomever forks it? Are there people that know enough about this lib that can support it going forward? There are currently 23 issues and 3 open pull requests.

It needs a resolution though, we just need to be sure that once it is forked, it will be supported by more than one person.

**Comment by Turbo87** [comment]
the other alternative would be to reactivate `cli-table` itself. @joshbetz apparently has merged some PRs over there and maybe he is interested in merging these two projects back together again? 🤔

**Comment by DanielRuf** [comment]
An org is definitely a good idea and would increase the truck factor.

**Comment by schalkneethling** [comment]
@Turbo87 @DanielRuf That is a great idea. I will definitely support the move to an org and getting a couple of maintainers to share the load. 👍

**Comment by Turbo87** [comment]
https://github.com/cli-table/cli-table3

you should get invites to the org in the next minutes :)

**Comment by Turbo87** [comment]
published the current `master` as `cli-table@0.4.0`

**Comment by ronkorving** [comment]
@Turbo87 You mean `cli-table3@0.4.0`?

**Comment by Turbo87** [comment]
whoops, yeah, that's what I meant to write

**Comment by schalkneethling** [comment]
@Turbo87 I guess we need to notify or open pull requests against packages that depends on cli-table2 to inform them of the new canonical repo? Thanks for getting this all set-up o/\o

**Comment by Turbo87** [comment]
yeah, that would a good idea. I've started by opening a PR at https://github.com/npm/npm/pull/20955. do you know other projects that use `cli-table2`?

**Comment by schalkneethling** [comment]
@Turbo87 Good question, let me see what some of my Snyk reports say. Also, we should probably set-up some kind off communication channel for the project. Perhaps Gitter or Slack?

**Comment by schalkneethling** [comment]
@Turbo87 I guess the list of dependants here would be projects that depend on `cli-table2` - https://www.npmjs.com/package/cli-table2

That is a *lot* of projects.

**Comment by DanielRuf** [comment]
455, we should create a list then =)

https://www.npmjs.com/package/cli-table2?activeTab=dependents

**Comment by schalkneethling** [comment]
Yup, and we should probably carry over the open issues from here to the new repo. Those that are still relevant. Also, there are 3 pull requests. If they are still relevant, we should probably ask the authors to open them against the new repo.

Thoughts?

**Comment by schalkneethling** [comment]
@DanielRuf @Turbo87 ok, there you go https://github.com/cli-table/cli-table3/issues/17 :)

**Comment by DanielRuf** [comment]
> DanielRuf @Turbo87 ok, there you go cli-table/cli-table3#17 :)

Probably good as list with checkboxes to see the status in the overview (todo list, x / y done).

**Comment by DanielRuf** [comment]
Let me know if I should help here (transforming the list).

**Comment by schalkneethling** [comment]
@DanielRuf @Turbo87 should we migrate the issues over from here to the new repo?

**Comment by Turbo87** [comment]
I'm not sure, some of them might not be relevant anymore...

**Comment by schalkneethling** [comment]
@Turbo87 very true, I was thinking if we want to triage it in some way? Perhaps look over them when we have time and add a comment with a 👍 or 👎
